config_test.cc

```diff
@@ -1,40 +1,40 @@
 #include "gtest/gtest.h"
+#include <string.h>
 #include "config.h"
 
 TEST(ServerConfigTest, SimpleConfig) {
   ServerConfig config("test_config_1");
   EXPECT_EQ(config.GetPortNo(), 12345);
+  std::cout << config.ToString() << "     " << "Server is running on Port: 12345";
+  EXPECT_EQ(0, config.ToString().compare(std::string{"Server is running on Port: 12345"}));
 }
 
 TEST(ServerConfigTest, PortTooLargeConfig) {
   try{
   	ServerConfig config("test_config_2");
-  	EXPECT_EQ(config.GetPortNo(), 80);
   }
   catch (PortRangeException &e)
   {
-	std::cout << e.what();
+	EXPECT_EQ(0, strcmp(e.what(), "Port number out of range."));
   }
 }
 
```

Comment:
Great test cases! We have covered whether the port used for the web server in the config file is out of range or if it is missing. It shows that our web server can handle invalid config files.